src/libqof/qof/qofbackend-p.h

```diff
@@ -354,6 +354,8 @@ extern "C"
  */
 void qof_backend_register_provider (/*@ only @*/ QofBackendProvider *);
 
+void qof_backend_unregister_provider (QofBackendProvider *);
```

Comment:
The only scenario I can think of for removing any backend providers is testing. It seems correct to only be able to remove the backend provider "you" put in there rather than anyone being able to remove all of them. Would you suggest removing this function, or changing it to `unregister_all_providers`?